bin/run-browser-tests

```diff
@@ -7,9 +7,7 @@ docker::set_project_name_browser_tests
 bin/pull-image
 
 docker run --rm -it \
-  -v "$(pwd)/browser-test/src:/usr/src/civiform-browser-tests/src" \
-  -v "$(pwd)/browser-test/bin:/usr/src/civiform-browser-tests/bin" \
-  -v "$(pwd)/browser-test/tmp:/usr/src/civiform-browser-tests/tmp" \
+  -v "$(pwd)/browser-test:/usr/src/civiform-browser-tests" \
```

Comment:
This all seems reasonable to me, though I'm still learning Docker as I go. @kmcnellis might be able to provide a more thorough review on correctness here than I.